fdbserver/TenantBalancer.actor.cpp

```
@@ -1839,8 +2004,18 @@ ACTOR Future<Void> cleanupMovementSource(TenantBalancer* self, CleanupMovementSo
 	    .detail("CleanupType", req.cleanupType);
 
 	try {
-		state Reference<MovementRecordMap::MutableRecord> mutableRecord =
-		    wait(self->mutateOutgoingMovement(req.prefix));
+		state ErrorOr<Reference<MovementRecordMap::MutableRecord>> mutableRecordResult = wait(errorOr(
+		    timeoutError(self->mutateOutgoingMovement(req.prefix), SERVER_KNOBS->TENANT_BALANCER_OPERATION_TIMEOUT)));
+		if (mutableRecordResult.isError()) {
```

If we remove the `errorOr`, we can just let this throw and handle it like all other errors.